src/runtime_src/core/tools/xbmgmt2/SubCmdConfigure.cpp

```diff
@@ -227,6 +227,33 @@ show_device_conf(xrt_core::device* device)
   std::cout << std::flush;
 }
 
+/*
+ * helper function for option:purge
+ * remove the daemon config file
+ */
+static void 
+remove_daemon_config()
+{
+  XBU::sudo_or_throw("Removing daemon configuration file requires sudo");
```

Comment:
Would it be helpful to the user to:
Prompt them indicating that the config_file is about to be removed.  This prompt should also include the name of the file.

Of course, if the --force flag is set then the prompt would automatically be performing a positive response..

Reply:
The common function to do this is canProceed()